### DISCUSSION TITLE
Error with plugin after update to SQLite studio 3.3.3

### DISCUSSION THREAD

**Discussion by pjfernan** [pawelsalawa/sqlitestudio#4235] [General]
### Details
Hello I have upgraded to 3.3.3 from a former version (I cannot figure out which one)
After succesfully installation and every time I open it, this error is shown:
"[06:31:06] Cannot load plugin DbSqliteCipher. Error details: Cannot load library /Applications/SQLiteStudio.app/Contents/PlugIns/libDbSqliteCipher.dylib: (dlopen(/Applications/SQLiteStudio.app/Contents/PlugIns/libDbSqliteCipher.dylib, 10): Symbol not found: ____chkstk_darwin
  Referenced from: /Applications/SQLiteStudio.app/Contents/PlugIns/../Frameworks/libcrypto.1.1.dylib (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib
 in /Applications/SQLiteStudio.app/Contents/PlugIns/../Frameworks/libcrypto.1.1.dylib)"

### Steps to reproduce
The error is shown after the app opens. I have tried to check/uncheck that plugin but not possible. Apps works fine in anyway. I don´t need such cipher plugin.

### Operating system
I am using Mac OS High Sierra. 10.13.6

I will appreciate the steps to remove the plugin reference to avoid such message.
Thanks.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
This plugin is no longer maintained in version 3.3.3, but I see you have them left from previous version. You can get rid of errors by going to the Configuration -> Plugins and there uncheck the SQLCipher plugin.

**Comment by pjfernan** [comment]
Hello pawelsalawa.
Such plugin was found unchecked. With no chance to check/uncheck. 
Anyway, after read you, I change any other option in /Preferences/Plugins and return it back to its original status, only to have the button "Apply" available. 

After that, the error dissapeared.

Thanks. Pablo.
